### PR TITLE
[#1145] Add to My Resources button no longer appears when not signed in

### PR DESCRIPTION
**Pull request by mauriel133**

### CODE REVIEW

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

File: theme/templates/resource-landing-page/top-right-buttons.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
                                 title="Remove from my resources" data-form-id="form-add-to-my-resources"
                                 class="glyphicon glyphicon-inbox icon-button btn-resource-remove"></button>
                     </form>
-                {% else %}
+                {% elif user.pk %}
```
Comment (pkdash) [inline theme/templates/resource-landing-page/top-right-buttons.html]: May be better to use ```user.is_authenticated``` instead of ```user.pk```

Comment (mauriel133) [inline theme/templates/resource-landing-page/top-right-buttons.html]: Fixed

**Comment by mjstealey** [comment]
+1

**Pull request merged**